docs/developer/metadata.mdx

```diff
@@ -55,9 +53,8 @@ type MetadataItem {
 
 Here's an example query that fetches the metadata of the first product:
 
-<GraphQL
-  height={25}
-  query={`{
+```graphql
```

Comment:
Would this be more readable as tabs?

Reply:
Tried this and IMO readability goes down - I prefer to see request and response the same time, it's easier for me to understand when I don't need to switch the tabs